sct-commons/src/test/java/org/lfenergy/compas/sct/commons/dto/ResumedDataTemplateTest.java

```diff
@@ -210,4 +212,28 @@ private static Stream<Arguments> daParametersProviderNotUpdatable() {
                 Arguments.of("should return false when Mod", CTL_MODEL, null, false)
         );
     }
+
+    @Test
+    void findFirstValue_should_return_first_found_value(){
+        // Given
+        ResumedDataTemplate rDTT = new ResumedDataTemplate();
+        DaTypeName da1 = createDa("da1", TFCEnum.CF, true, Map.of(0L, "a value"));
```

Comment:
Map might have two values to see what the ethod do clearly

Reply:
OK